pep-8100.rst

```diff
@@ -50,6 +50,65 @@ Once the nomination period opens, candidates will be listed here:
 1. *Candidate (nominated by)*
 
 
+Voter Roll
+==========
+
+All CPython core developers are eligible to vote. Ballots will be distributed
```

Comment:
This is perfectly fine, but I wonder if we want to be even more pedantic and use the language from PEP 8016/13? The technical rules are "All active Python core team members are eligible to vote". Since we haven't added any new members this month, "active core team member" means everyone on the github team (i.e., exactly the same people who could vote in the last election), plus any past core devs who were inactive but decide to mark themselves "active" again.

(Everything else looks great to me.)

Reply:
👍